src/UserFunctions.java

```diff
@@ -12,7 +13,9 @@ public UserFunctions() {
     }
 
     /** The path to the dictionary file for the game. */
-    public static final String DICTIONARY_PATH = "dictionary.txt";
+    public static final String DICTIONARY_PATH = "src\\dictionary_optimized.txt";
+    /** The path to the dictionary of valid starting words. */
+    public static final String ROOT_DICTIONARY_PATH = "src\\dictionary_optimized.txt";
```

Comment:
Wouldn't this one be "src\\dictionary_roots.txt"? It'll still work with dictionary_optimized, it'll just be a bit slower.